smoke/srv6_test.sh

```diff
@@ -0,0 +1,295 @@
+    grcli add sr policy bsid 5f00::100 seglist 5f00::1 5f00::2 5f00::3
+    grcli add sr policy bsid 5f00::101 seglist 5f00::10 weight 2
+    grcli add sr policy bsid 5f00::102 seglist 5f00::1 5f00::2 5f00::3 fe80::d0ad:caff:fefe:b412 fe80::d0ad:caff:fefe:b413 fe80::d0ad:caff:fefe:b414
+    grcli add sr policy bsid 5f00::103 seglist 5f00::10 5f00::11 encap h.encaps.red weight 3
+    grcli add sr policy bsid 5f00::104 seglist 5f00::12 encap h.encaps.red
+    grcli add sr policy bsid 5f00::105 seglist 5f00::12
+    grcli add sr policy bsid 5f00::100 seglist 5f00::1 5f00::2 5f00::3 && fail "bsid already exist"
+    grcli add sr steer 10.0.0.0/16 bsid 5f00::99 && fail "bsid not existing"
+    grcli add sr steer 10.0.0.0/16 bsid 5f00::100
+    grcli add sr steer 100::/64 bsid 5f00::100
+    grcli add sr steer 10.1.0.0/16 bsid 5f00::101
+    grcli add sr steer 10.1.0.0/16 bsid 5f00::102
+    grcli add sr steer 10.1.0.0/16 bsid 5f00::103
+    grcli add sr steer 10.1.0.0/16 bsid 5f00::104
+    grcli add sr steer 10.1.0.0/16 bsid 5f00::105
+    grcli show sr policy
+    grcli show sr steer
+    grcli del sr policy bsid 5f00::100
+    grcli del sr steer 10.0.0.0/16 && fail "no entry"
+    grcli del sr steer 10.1.0.0/16
+    grcli show sr policy
+    grcli show sr steer
+    grcli add sr steer 0.0.0.0/0 bsid 5f00::104
+    grcli add sr steer ::/0 bsid 5f00::105
+
+    grcli add sr localsid 1::2 behavior end.dt4
+    grcli add sr localsid 2::2 behavior end.dt4 table 25
+    grcli add sr localsid 3::2 behavior end
+    grcli add sr localsid 3::2 behavior end vrf 50
+    grcli add sr localsid fe80::d0ad:caff:fefe:b412 behavior end.t table 100 vrf 50
+    grcli show sr localsid
+    grcli del sr localsid 3::2
+    grcli del sr localsid 3::2 && fail "no entry"
+    grcli del sr localsid 3::2 vrf 50
+    grcli show sr localsid
+}
+
+setup_ns() {
+    ip netns del $p0 2> /dev/null || true
+    ip netns del $p1 2> /dev/null || true
+    ip netns add $p0
+    ip netns add $p1
+
+    # setup ports and connected
+    grcli add interface port $p0 devargs net_tap0,iface=$p0 mac d2:f0:0c:ba:a5:10
+    grcli add interface port $p1 devargs net_tap1,iface=$p1 mac d2:f0:0c:ba:a5:11
+    grcli add ip6 address fd00:101::1/64 iface $p0
+    grcli add ip6 address fd00:102::1/64 iface $p1
+    grcli add ip address 192.168.61.1/24 iface $p0
+    grcli add ip address 192.168.62.1/24 iface $p1
+    ip link set $p0 netns $p0
+    ip -n $p0 link set $p0 address d2:ad:ca:fe:b4:10
+    ip -n $p0 link set lo up
+    ip -n $p0 link set $p0 up
+    ip -n $p0 addr add fd00:101::2/64 dev $p0
+    ip -n $p0 addr add 192.168.61.2/24 dev $p0
+    ip link set $p1 netns $p1
+    ip -n $p1 link set $p1 address d2:ad:ca:fe:b4:11
+    ip -n $p1 link set lo up
+    ip -n $p1 link set $p1 up
+    ip -n $p1 addr add fd00:102::2/64 dev $p1
+    ip -n $p1 addr add 192.168.62.2/24 dev $p1
+}
+
+send_py_pkt() {
+    port=$1
+    script_head="
+#!/usr/bin/env python3
+from scapy.all import *
```

Comment:
Make sure to add python3-scapy to the dev dependencies in `README.md` and in `.github/workflows/*`.